src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -164,7 +164,7 @@ namespace winrt::TerminalApp::implementation
         winrt::fire_and_forget _CopyToClipboardHandler(const IInspectable sender, const winrt::Microsoft::Terminal::TerminalControl::CopyToClipboardEventArgs copiedData);
         winrt::fire_and_forget _PasteFromClipboardHandler(const IInspectable sender,
                                                           const Microsoft::Terminal::TerminalControl::PasteFromClipboardEventArgs eventArgs);
-        bool _CopyText(const bool trimTrailingWhitespace);
+        bool _CopyText(const bool singleLine = false, const Windows::Foundation::IReference<Microsoft::Terminal::Settings::CopyFormat> formats = nullptr);
```

Comment:
do we ever call this without providing all 2 parameters?

you've got _two default parameters_, which means it can be called as...

```
_CopyText()
_CopyText(true)
_CopyText(false, nullptr)
_CopyText(false, value)
```

Reply:
_yeaaaah_ this code was around before you told me how default params are not as _amazing_ as I like. Removing haha.

Just so you know, an example like this is exactly _why_ I think default params are the coolest thing _ever_. Why define 3 method signatures, when you can just define 1 with 2 back-to-back default params! :P